### ISSUE TITLE
Show timezone by default

### ISSUE THREAD

**Issue by sindresorhus** [SamVerschueren/dev-time-cli#5]
From https://github.com/SamVerschueren/dev-time-cli/commit/de77c2b7d56121a148638b0b1e91a5e8cc45678d#commitcomment-14815063.

> It might be useful to include the timezone too, so I don't have to calculate it in my head.
> I would also go with `07` => `7`.
> And the time first.

**Comment by SamVerschueren** [comment]
Ah, missed that one, thanks. Which one do you prefer?

1. 13:18:08 - 8 Dec. 2015 - +01:00
2. 13:18:08 - +01:00 - 8 Dec. 2015
3. 13:18:08 +01:00 - 8 Dec. 2015

---------
I think it's difficult to provide a nice default for everyone. Personally I'm not really interested in the timezone of the user but just want to know the local time. So for me personally, the default is fine as it is. Maybe we can use `configstore` to use the previously used `--format` as default? Just asking your opinion on this one :).

**Comment by sindresorhus** [comment]
1., but `13:18 - 8 Dec. 2015 - +1`

**Comment by SamVerschueren** [comment]
What about timezones with half an hour? `+5:30`?

That output screws up with my `--format` flag though, it accepts a `moment` format. Outputting the timezone as `+1` is not one of the options :).

**Comment by sindresorhus** [comment]
> What about timezones with half an hour? +5:30?

Wut, that's a thing? I thought timezones were integers.

**Comment by sindresorhus** [comment]
Maybe add a `--verbose` mode to the default for those that just want all the info without having to manually format it?

**Comment by sindresorhus** [comment]
I also think the seconds in the time is pretty useless and should be removed.

**Comment by SamVerschueren** [comment]
> Wut, that's a thing? I thought timezones were integers.

Jup, that's a thing :) 

> I also think the seconds in the time is pretty useless and should be removed.

:+1:

**Comment by SamVerschueren** [comment]
@sindresorhus So to wrap things up in this feature request(s).

### remove seconds and put time first

```
$ dev-time sindresorhus
19:17 - 08 Dec. 2015
```

### Add `--verbose` (`-v`) flag for more information

```
$ dev-time sindresorhus --verbose
19:17 - 08 Dec. 2015 - +1
```

Adding `+1` might be ambiguous. Is it `+1` from your current timezone or is it `+1` from GMT? So maybe we should write it as `19:17 - 08 Dec. 2015 - GMT+1`.

What should we do with the timezone `+05:30`? --> `GMT+5:30`?

---------
Did I miss something here?

**Comment by sindresorhus** [comment]
:+1: But `UTC+1`

**Comment by SamVerschueren** [comment]
:+1:

**Comment by sindresorhus** [comment]
UTC+5.5

**Comment by SamVerschueren** [comment]
Isn't that weird :D?

**Comment by sindresorhus** [comment]
Not for me, it's an offset, not a time. But UTC+5:30 is fine too if you rather have that.

**Comment by SamVerschueren** [comment]
Released in `v1.2.0`. Thanks for the feedback! Much appreciated. :beers:
